File: Home Light Automation/workspace/iot-device/apps/project/GeolocationData.py
'''
Created on 26-Jan-2020

@author: soman
'''

class GeolocationData():

#   Define variable for Latitude value of current location
    lat = 0.0;

#   Define variable for Longitude value of current location     
    lng = 0.0;
    
#   Define variable for Timezone of current location     
    timezone = None;

#   Define GeolocationData constructor
    def __init__(self): 
        super()
        
#   Define getter method for Latitude
    def getlat(self):
        return self.lat

#   Define getter method for Longitude    
    def getlng(self):
        return self.lng
 
#   Define setter method for Latitude    
    def setlat(self, lat):
        self.lat = lat
 
#   Define setter method for Longitude    
    def setlng(self, lng):
        self.lng = lng 
        
#   Define getter method for Longitude    
    def gettimezone(self):
        return self.timezone
 
#   Define setter method for Latitude    
    def settimezone(self, timezone):
        self.timezone = timezone

#    String representation of GeolocationData object
    def __str__(self):
        String_obj = "Latitude=" + str(self.lat) + "," + "Longitude=" + str(self.lng) + "," + "Timezone=" + self.timezone
        return String_obj  
    

File: Home Light Automation/workspace/iot-gateway/src/main/java/neu/s/dey/connecteddevices/project/GeolocationData.java
package neu.s.dey.connecteddevices.project;

import java.io.Serializable;
import java.util.Date;
import java.sql.Timestamp;

public class GeolocationData implements Serializable	{

//	Define variable for Latitude value of current location
	public double lat = 0.0;

//	Define variable for Longitude value of current location 	
	public double lng = 0.0;

//	Define variable for Timezone value of current location	
	public String timezone = null;
	
//	Define GeolocationData Constructor
	public GeolocationData()	{
		super();
	}

// Getter and Setter methods for lat, lng and timezone
	public double getlat() {
		return lat;
	}

	public void setlat(double lat) {
		this.lat = lat;
	}

	public double getlng() {
		return lng;
	}

	public void setlng(double lng) {
		this.lng = lng;
	}

	public String gettimezone() {
		return timezone;
	}

	public void settimezone(String timezone) {
		this.timezone = timezone;
	}
	
// String representation of GeolocationData object	
	public String toString() {
	       String String_obj = 
	          "lat=" + String.valueOf(this.getlat()) + "," + 
	          "lng=" + String.valueOf(this.getlng()) + "," +        
	       	  "timezone=" + String.valueOf(this.gettimezone());	        
	           return String_obj; 
	}

}